neo/Ledger/RelayResultReason.cs

```diff
@@ -7,6 +7,7 @@ public enum RelayResultReason : byte
         OutOfMemory,
         UnableToVerify,
         Invalid,
+        PolicyFail,
```

Comment:
I think it's important to given explicit numbers here. Example: Invalid = 0x05.
Because after adding new options, the older ones may change byte identifier (and afterall, the byte is what really matters to developers).

Reply:
That is true, broda!